app/javascript/react/screens/App/Plan/PlanActions.js

```diff
@@ -206,3 +211,81 @@ export const downloadLogAction = task => dispatch => {
     })
   });
 };
+
+// *****************************************************************************
+// * CANCEL MIGRATION TASKS
+// *****************************************************************************
+export const cancelPlanRequestTasksAction = (url, tasks) => dispatch => {
+  const completedTasks = [];
+  const incompleteTasks = [];
+  tasks.map(t => (t.completed ? completedTasks.push(t) : incompleteTasks.push(t)));
```

Comment:
While this ultimately has the desired effect, I think we may want to use `forEach` here.  In my experience, we use `map` when we want to do something with the returned array.  In this case, it seems that we are iterating through `tasks` (rather than mapping it to a new array)

Reply:
Beat me to it!

Reply:
Addressed in the last commit